JavaAssignments/src/rahulV/Assignment2_Q6.java

```diff
@@ -0,0 +1,39 @@
+package rahulV;
+
+public class Assignment2_Q6 {
+
+	void CheckEqualityofTwoArrays(int array1[], int array2[]) { //method to check equality of two arrays
+		boolean flag = true;
```

Comment:
Logic is correct but you can make this code more clean by following below code style.

if(array1.length != array2.length)
         return false;
else
       // write your comparison logic here.